gibberish_server/app.js

```diff
@@ -8,7 +8,7 @@ require('dotenv').config()
 
 const app = express();
 const server = http.createServer(app);
-const PORT = 3000;
+const PORT = 4000;
```

Comment:
why change this again to 4000?

Reply:
because react is in 3000

Reply:
@ryanmt95 do you have any objection since you are the one who change it just on the previous pr?

Reply:
nope, 4000 is fine